index.html

```diff
@@ -72,8 +72,199 @@ <h1 class="intro">
+            <div class="pale-info">JavaScript</div>
+          </div>
+          <div class="action">
+            <button class="see-project" type="button">Project</button>
+          </div>
+        </div>
+      </div>
+    </section>
+
+    <!-- About section -->
+    <section class="about-section">
+      <div class="top">
+        <h1 class="intro">About Myself</h1>
+
+        <p class="description-container">
+          I'm a software developer! I can help you build a product , feature or
+          website. Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don't hestiate to contact
+          me.
+        </p>
+        <span class="connect">LET'S CONNECT</span>
+        <div class="social">
+          <ul>
+            <li>
+              <a href="#"><span class="fa fa-twitter twitter"></span></a>
+            </li>
+            <li>
+              <a href="#"><span class="fa fa-linkedin"></span></a>
+            </li>
+            <li>
+              <a href="#"><span class="fa fa-medium"></span></a>
+            </li>
+            <li>
+              <a href="#"><span class="fa fa-github"></span></a>
+            </li>
+            <li>
+              <a href="#"><span class="fa fa-hand-peace"></span></a>
+            </li>
+          </ul>
+        </div>
+        <div class="for-space"></div>
+      </div>
+      <div class="action">
+        <button class="see-project" type="button">Get My Resume</button>
+      </div>
+
+      <!-- For skills section -->
+      <section>
+        <ul class="main-list">
+          <!-- skill item 1 -->
+          <li class="skill-item">
+            <div class="skill-headline">
+              <div>
+                <h3>Languages</h3>
+              </div>
+              <div>
+                <a href="#"
+                  ><img
+                    class="icon"
+                    src="./Assets/dropdown-icon.png"
+                    alt="drop_down_icon"
+                /></a>
+              </div>
+            </div>
+            <ul>
+              <li>
+                <img
+                  src="./Assets/js-logo.png"
+                  class="skill-logo"
+                  id="js-logo"
+                  alt="js_logo"
+                />
+                <div class="skill-text">JavaScript</div>
+              </li>
+              <li>
+                <img
+                  src="./Assets/html_logo.png"
+                  class="skill-logo"
+                  alt="html_logo"
+                />
+                <div class="skill-text" id="skill-text-special_1">Html</div>
+              </li>
+              <li>
+                <img
+                  src="./Assets/css_logo.png"
+                  class="skill-logo"
+                  id="css-logo"
+                  alt="css_logo"
+                />
+                <div class="skill-text" id="skill-text-special_2">Css</div>
+              </li>
```

Comment:
- [ ] The width of `li`s should be 100%. Kindly fix it 😃 
- This is your design:
<img width="184" alt="image" src="https://user-images.githubusercontent.com/77190203/148648353-01daae8a-3700-40ad-a2a2-cb0200eada68.png">
- Figma design:
<img width="176" alt="image" src="https://user-images.githubusercontent.com/77190203/148648398-50e57b06-61e1-4055-94d4-a6c15c391240.png">